tests/seeding/test_find_or_create.py

```diff
@@ -349,7 +279,6 @@ def test_create_or_update_unique_found(predictor_collection):
                             latent_variables = [])
     #verify that the returned object is updated
     returned_pred = create_or_update(collection=predictor_collection, resource=pred)
-    assert returned_pred.uid == pred.uid
```

Comment:
The behavior of the non-fake client is that it doesn't modify the object that was passed in, so this was testing fake behavior that's inconsistent with the real behavior.